src/Core/Content/Product/SalesChannel/CrossSelling/ProductCrossSellingRoute.php

```diff
@@ -68,8 +68,8 @@ public static function buildName(string $id): string
     #[Route(
         path: '/store-api/product/{productId}/cross-selling',
         name: 'store-api.product.cross-selling',
-        defaults: ['_entity' => 'product'],
-        methods: ['POST']
+        methods: [Request::METHOD_POST, Request::METHOD_GET],
+        defaults: [PlatformRequest::ATTRIBUTE_ENTITY => 'product', PlatformRequest::ATTRIBUTE_HTTP_CACHE => true]
```

Comment:
```suggestion
        defaults: [PlatformRequest::ATTRIBUTE_ENTITY => \Shopware\Core\Content\Product\ProductDefinition::ENTITY_NAME, PlatformRequest::ATTRIBUTE_HTTP_CACHE => true]
```

if you want to take it to the top :sweat_smile:

Reply:
Done